src/outputs/browserextension_tour.js

```diff
@@ -7,7 +8,7 @@
 				'mediawiki.jqueryMsg'
 			] )
 		).then( function () {
-			if ( !$( '#t-whowrotethat' ).length ) {
+			if ( !$( '#t-whowrotethat' ).length || !activationInstance.isValidPage() ) {
```

Comment:
I'm a bit confused. Aren't we already using `isValidPage` to determine if we add `#t-whowrotethat`? So it already shouldn't be present on invalid pages.